src/udev/udev-watch.c

```diff
@@ -12,8 +12,13 @@
 #include "dirent-util.h"
 #include "fs-util.h"
 #include "parse-util.h"
+#include "random-util.h"
 #include "udev-watch.h"
 
+#define SAVE_WATCH_HANDLE_MAX_RETRIES  128
+#define MAX_RANDOM_DELAY (250 * USEC_PER_MSEC)
+#define MIN_RANDOM_DELAY ( 50 * USEC_PER_MSEC)
```

Comment:
In some cases 50ms is a lot and measurable when profiling, can this be a bit smaller? 10?

Reply:
OK. Now, the range is 10-100 msec.